example/spec/parallel/ingestGranule/IngestUMMGSuccessSpec.js

```diff
@@ -213,6 +213,12 @@ describe('The S3 Ingest Granules workflow configured to ingest UMM-G', () => {
 
   afterAll(async () => {
     // clean up stack state added by test
+    await Promise.all(inputPayload.granules.map(
+      (g) => granulesApiTestUtils.removePublishedGranule({
```

Comment:
Nit: this is deprecated, we should be using the api-client methods directly

Reply:
Definitely agree, thank you for this call out. I will handle this in this PR and comment back with the commit 👍

Reply:
Sure thing -- to be clear I'm fine with punting this to scoped work/a ticket to wholesale remove these calls (they're all over in our testing).